skorch/net.py

```diff
@@ -1524,6 +1534,90 @@ def __setstate__(self, state):
 
         self.__dict__.update(state)
 
+    def _register_attribute(
+            self,
+            name,
+            prefixes=True,
+            cuda_dependent_attributes=True,
+    ):
+        """Add attribute name to prefixes_ and
+        cuda_dependent_attributes_.
```

Comment:
This should include the purpose of registering an attribute.

Reply:
done